geomstats/geometry/special_euclidean.py

```diff
@@ -7,7 +7,7 @@
 import geomstats.backend as gs
 import geomstats.vectorization
 from geomstats.geometry.euclidean import Euclidean
-from geomstats.geometry.general_linear import GeneralLinear
+from geomstats.geometry.general_linear import GeneralLinear, Matrices
```

Comment:
why not: `from geomstats.geometry.matrices import Matrices`? Written like this it seems that `Matrices` is in the general_linear file? Is it because you were dealing with circular imports?

Reply:
No just to avoid adding a line...